docs/rnm-getting-started.md

```diff
@@ -5,7 +5,7 @@ title: Get Started with macOS
 
 This guide will help you get started on setting up your very first React Native for macOS app.
 
->** Latest stable version available for React Native for macOS is 0.71**
+>** Latest stable version available for React Native for macOS is 0.75**
```

Comment:
This won't show up on the website. You'll also need to edit (create) the 0.75 versioned doc of this file. You'll also probably want to fix the "old" new project commands (`react-native@latest init`) since that won't work anymore.